core-plugins/src/main/java/co/cask/hydrator/plugin/batch/sink/S3AvroBatchSink.java

```diff
@@ -111,6 +117,19 @@ public S3AvroOutputFormatProvider(S3AvroSinkConfig config, BatchSinkContext cont
       conf = Maps.newHashMap();
       conf.put(JobContext.OUTPUT_KEY_CLASS, AvroKey.class.getName());
       conf.put("avro.schema.output.key", config.schema);
+      if (config.compressionCodec != null && !config.compressionCodec.equals("None")) {
+        conf.put("mapred.output.compress", "true");
```

Comment:
Can we put "mapred.output.compress" and such in a static variable at the top of the class.